requirements.in

```diff
@@ -49,6 +49,6 @@ azure-mgmt-resourcegraph==8.0.0
 pydantic==1.10.8
 azure-storage-blob==12.16.0
 questionary==1.10.0
-pbr==1.4
```

Comment:
Why did we need to upgrade this one ?

Reply:
Can't build a docker image without it :(

Reply:
let's remove it from requriements.in, `hdrhistogram` is using it, not SCT directly.

Reply:
Can't rebuild a Docker image right now without updating PyYAML to 6.0.1 and awscli to >=1.29.4 because of the problem with fresh Cython.

Reply:
Why in the first place we needed upgrade of Google tools ? What wasn't working ?

Reply:
Why in the first place we needed upgrade of Google tools ? What wasn't working ?

Reply:
o.k. next one that would try to build a new image, is heads for a surprise
can you open an issue with the details of what you do know, and which version are currently broken ?

Reply:
Sure, I'll do.

Reply:
@enaydanov please add a link to the issue here. thanks

Reply:
https://github.com/scylladb/scylla-cluster-tests/issues/6389